corehq/apps/accounting/bootstrap/utils.py

```diff
@@ -112,9 +107,8 @@ def _ensure_features(edition, verbose, apps):
         log_accounting_info(f"Ensuring Features for plan: {edition}")
 
     features = []
-    for feature_type in FEATURE_TYPES:
-        # Don't prefix web user feature name with edition
-        if feature_type == FeatureType.WEB_USER:
+    for feature_type in feature_rates.keys():
+        if feature_type not in FeatureType.EDITIONED_FEATURES:
```

Comment:
i know the original line was added when the `WEB_USER` feature was introduced, but to me it always makes more sense for readability to have a true statement for the limited set, and then else catch everything else. I would rewrite this block to instead be

```
if feature_type in FeatureType.EDITIONED_FEATURES:
    feature = Feature(name=f"{feature_type} {edition}", feature_type=feature_type)
else:
    feature = Feature(name=feature_type, feature_type=feature_type)
```

Reply:
I went a slightly different route after noticing we were instantiating a Feature whether or not it was necessary. Still took the idea of using an `if` rather than an `if not`. https://github.com/dimagi/commcare-hq/pull/34847/commits/88b54f83f7df774b88b02b48d8f0a1aaf6d512af